src/Extend/SitesLinkedToMagentoStores.php

```diff
@@ -12,12 +13,15 @@ public function findByUrl($url)
             return $site;
         }
 
-        return parent::findByUrl($url);
+        return Cache::store('array')->rememberForever('rapidez.statamic.findByUrl-'.$url, fn() => parent::findByUrl($url));
```

Comment:
So this is really faster? With or without; both will get data from an array eventually.